### PR TITLE
SLUDGE: Add detection for v 1.0 outoforder 

### PR DESCRIPTION
**Pull request by oknowpressspacetojump**
I've double checked the version, and did not found v0.1

### CODE REVIEW

File: engines/sludge/detection_tables.h [hunk 1]
```diff
@@ -114,6 +114,20 @@ static const SludgeGameDescription gameDescriptions[] = {
 		0
 	},
 
+	{
+		{
+			"outoforder",
+			"",
```
Comment (sev-) [inline engines/sludge/detection_tables.h:120]: Please specivy the version here

File: engines/sludge/detection_tables.h [hunk 2]
```diff
@@ -114,6 +114,20 @@ static const SludgeGameDescription gameDescriptions[] = {
...
+			AD_ENTRY2s("gamedata",         "4d72dbad0ff170169cd7e4e7e389a90d", 21122647,
+					   "Out Of Order.exe", "ebc386dd0cb77df68dd12d72742eb310", 68608),
+			Common::EN_ANY,
+			Common::kPlatformUnknown,
```
Comment (sev-) [inline engines/sludge/detection_tables.h:124]: Please set it to Windows

**Review by sev- — CHANGES_REQUESTED** [review]
Please fix two little thing and change the commit log message to be specific, which game you added, like: "SLUDGE: Added detection for v 0.1 of outoforder"

By changing the commit, DO NOT CLOSE this PR. Rather just force-push and GitHub will pick it up.

**Review by oknowpressspacetojump — COMMENTED** [review]
fixed. added version for the pervious entry as well

**Comment by sev-** [comment]
Thanks!

**Pull request merged**